pages/plugins/writing.md.erb

```diff
@@ -215,6 +215,23 @@ Next, add a `README.md` file to introduce the plugin to the world:
 
 <%= render 'plugins/tutorial_readme' %>
 
+## Developing a plugin with a feature branch
+
+When developing your plugin, it might be useful to have a quick feedback loop between making a change in your plugin code, and seeing the effects in a Buildkite pipeline.  Let's say you're developing your feature on `my-org/plugin#dev-branch`.  By default, if a Buildkite agent sees that it needs the plugin `my-org/plugin#dev-branch`, and it already has a checkout matching that spec, it will *not* pull any Git changes.  There's a handy feature for when you're developing on a plugin branch and you *do* want to see changes reflected immediately: the configuration setting [`plugins-always-clone-fresh`](/docs/agent/v3/configuration#plugins-always-clone-fresh).
```

Comment:
```suggestion
When developing plugins, it is useful to have a quick feedback loop between making a change in your plugin code, and seeing the effects in a Buildkite pipeline. Let's say you're developing your feature on `my-org/plugin#dev-branch`. *By default*, if a Buildkite agent sees that it needs the plugin `my-org/plugin#dev-branch`, and it already has a checkout matching that, it will *not* pull any changes from the Git repository. But if you *do* want to see changes reflected immediately, set [`plugins-always-clone-fresh`](/docs/agent/v3/configuration#plugins-always-clone-fresh) to `true`.
```

Reply:
Yep, yours is way better – thank you!